app/scripts/lib/oauth-errors.js

```diff
@@ -47,7 +47,7 @@ define(function (require, exports, module) {
       errno: 108,
       message: t('Invalid token')
     },
-    INVALID_REQUEST_PARAMETER: {
+    INVALID_PARAMETER: {
```

Comment:
All the other error modules use `INVALID_PARAMETER`, so this was made consistent.